src/config/mod.rs

```diff
@@ -156,8 +156,10 @@ stderr_file = "/tmp/psh.stderr"
 working_directory = "/"
 
 [rpc]
-addr = "www.optimatist.com"
+enable = true
```

Comment:
It's a good idea to split "make rpc switchable" and "add heartbeat time" into two commits cuz it makes your commit history cleaner and easier to review.